app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def create
+    @job = Job.find(params[:job_id])
+    @comment = @job.comments.create(comment_params)
+
+    redirect_to company_job_path(@job.company_id, @job)
+  end
+
+private
+
+  def comment_params
+    params.require(:comment).permit(:content, :job_id)
```

Comment:
we dont really need a `job_id` permitted because it is coming in our params and we are using `params[:job_id]` to find the job and then attach the comment to it.